apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -791,8 +792,8 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
             // this snapshot is never stored.
             _vote.quietEndingSnapshotSupport = _voterStateFor(_wasAccepted);
         } else {
-            // We only store the quiet ending extension if it wasn't confirmed yet. Note that we are trusting `_canVote()` for that.
-            // If we reached this point, it means the vote's flip was confirmed but the last computed end date is in the past.
+            // We are calculating quiet ending extensions via "rolling snapshots", and so we only update the vote's cached duration once
+            // the last period is over and we've confirmed the flip.
```

Comment:
@facuspagnuolo I was still a bit confused by the old comment, so I took the liberty of rewriting it.